tensornetwork/tests/network_components_free_test.py

```diff
@@ -0,0 +1,806 @@
+import numpy as np
+import tensorflow as tf
+import pytest
+from collections import namedtuple
+import h5py
+from tensornetwork.network_components import Node, CopyNode, Edge
+import tensornetwork as tn
+
+tf.compat.v1.enable_v2_behavior()
```

Comment:
Remove this as it is not needed after https://github.com/google/TensorNetwork/pull/263

Reply:
done